skins/oasis/modules/OasisController.class.php

```diff
@@ -158,9 +152,6 @@ public function executeIndex($params) {
 		$this->itemType = self::getItemType();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)